src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -204,7 +204,7 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 		}
 
 		// if prompt attached as an implicit "current file" context
-		return (this.implicitContext.isPrompt && this.implicitContext.enabled);
+		return (this.implicitContext.isInstructions && this.implicitContext.enabled);
```

Comment:
This change won't make sense - the `hasInstructionAttachments` is used to set the context that enables/disables the "submit" button of the chat, but it should only be enabled for "prompts" but not for "instructions".

This also causes instructions to be rendered with the "Current prompt" label in the chat input.